### ISSUE TITLE
Scatterplot with CSV with null values shows null as 0

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#6615]
```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "data": {"url": "data/penguins_size.csv"},
  "mark": "point",
  "encoding": {
    "x": {
      "field": "body_mass_g",
      "type": "quantitative",
      "scale": {"zero": false}
    },
    "y": {
      "field": "flipper_length_mm",
      "type": "quantitative",
      "scale": {"zero": false}
    },
    "color": {"field": "species", "type": "nominal"},
    "shape": {"field": "species", "type": "nominal"}
  }
}
```

![image](https://user-images.githubusercontent.com/589034/84620309-7af8c680-ae8c-11ea-8017-f1aa489b5b19.png)

```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "data": {
    "url": "data/movies.csv"
  },
  "layer": [
    {
      "mark": {
        "type": "point",
        "filled": true
      },
      "encoding": {
        "x": {
          "field": "Rotten Tomatoes Rating",
          "type": "quantitative"
        },
        "y": {
          "field": "IMDB Rating",
          "type": "quantitative"
        }
      }
    },
    {
      "mark": {
        "type": "line",
        "color": "firebrick"
      },
      "transform": [
        {
          "loess": "IMDB Rating",
          "on": "Rotten Tomatoes Rating"
        }
      ],
      "encoding": {
        "x": {
          "field": "Rotten Tomatoes Rating",
          "type": "quantitative"
        },
        "y": {
          "field": "IMDB Rating",
          "type": "quantitative"
        }
      }
    }
  ]
}
```

![image](https://user-images.githubusercontent.com/589034/84814017-b34ff000-afc5-11ea-8ff4-64a1cf7f7977.png)

**Comment by domoritz** [comment]
Related pull request https://github.com/vega/vega-lite/pull/4993

**Comment by ChiaLingWeng** [comment]
It's already fixed at v5.17.0
[Open the Chart in the Vega Editor](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAKyUAVhDYA7EABoQAEySYUqUAwBOgtCrVICCNsRpwIUmfIC+SwUgCecbWgDaoZNoDWaUJkccOH0cNhpZTEUQADMaQUE4ZTRMbQY4OxA4WSg2ZXCyPxAAD0LYuEEk9AAlNmwsgAIAFTZkTDYLeqq1fKiAoP0ARwYkCLpu0hAMx1LzCv0ASQBZABEAIU7u2QKlPuD0IZHMMaOJmzOFTyQfQt39YVlgpRzBNld0WO04Jm0aKF8MlIjCDRV4IdygF4WCALFbrLpHLZROT6Gp1WRNFpqdoQDYIgo2AC6SiyOTyiM0xRm5UqIFRmAazVa2NxPR2gT2IAOo3UJ2CUypc3QSzWLMRbP6+2G3PGfLOBJsQA)
